stable/weave-scope/README.md

```diff
@@ -0,0 +1,65 @@
+# Weave Scope
+
+## About this chart
+
+This chart contains two subcharts (*weave-scope-frontend* and *weave-scope-agent*, aliased as *frontend* and *agent* respectively) which deploy the corresponding components of Weave Scope, an interactive container monitoring and visualization application.
+
+Either subchart can be deployed on its own or the two can be deployed together (the default).
+
+## Prerequisites
+
+* This chart is designed and tested with Weave Scope 1.6.2 and Kubernetes 1.7.
+* Weave Scope 1.6.2 was actually released by WeaveWorks for Kubernetes 1.6 but seems to work fine on 1.7.
+* On Kubernetes 1.6 Weave Scope versions as old as 1.3.0 will probably work.
+* The service account, cluster role, cluster role binding and service specified in the rendered version of this chart must not already exist.
+
+## Values templated in this chart
+
+Note that most of this documentation is repeated in `values.yaml`; if you're in a hurry you can skip this part here and read it there.  Values with no default noted have no default.
+
+### Global values
+
+| Value | Default | Explanation |
+|------:|:--------|:------------|
+| **image** | "weaveworks/scope" | the image that will be used for this release |
+| **imageTag** | 1.6.2 | the version of Weave Scope desired for this release |
+| **service.name** | "weave-scope-app" | the short name desired for the frontend service -- this is a global so we can access its value easily from the agent subchart |
+| **service.port** | 80 | the port exposed by the Scope frontend service -- this is a global so we can access its value easily from the agent subchart |
+| **service.type** | "ClusterIP" | the frontend service type must be ClusterIP, NodePort or LoadBalancer -- this is a global to keep it with the other values for configuring the frontend service |
+| **global.rbac** | true | whether RBAC is enabled in the cluster or not |
+
+
+### Weave Scope frontend values
+
+The **weave-scope-frontend** section controls how the Scope frontend is installed.
+
+| Value | Default | Explanation |
```

Comment:
Please go with the style that is commonly used, which is 
`| Parameter | Description | Default |`.